_posts/2021-01-12-december-rubygems-update.md

```diff
@@ -0,0 +1,46 @@
+---
+title: December 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in December.
+
+## RubyGems News
+In December, we released `bundler 2.2` and `rubygems 3.2` 🎉. We implemented a unified release workflow and fixed a longstanding gem install issue. - [#4110](https://github.com/rubygems/rubygems/pull/4110)
+
+We killed a couple of regressions in `bundler 2.2.0` and `rubygems 3.2.0` and released a couple of patch level versions for each of them. 
+
+Finally, we fixed a gem command that was crashing or asking for OTP twice when it prompted for an email and password. - [#4162](https://github.com/rubygems/rubygems/pull/4162)
```

Comment:
I would rewod the first three paragraps to something like this:

> In December, we finally released `bundler 2.2` and `rubygems 3.2` 🎉. We also received the corresponding feedback and regression reports,and addressed almost everything reported through 4 patch level releases of each library. In particular, we made it on time for Ruby's Christmas release and managed to include rubygems 3.2.3 and bundler 2.2.3 with the final release of ruby 3.0.

> To be mentioned apart, together with rubygems 3.2.3 we released a fix to a longstanding gem install issue. From now on, `gem install` will respect the running ruby version, and install the latest version of the target gem that supports it. See [#4110](https://github.com/rubygems/rubygems/pull/4110) for more information.

Reply:
great thanks! I'll add the update.